aria/orchestrator/execution_plugin/ctx_proxy/server.py

```diff
@@ -150,43 +150,101 @@ def __exit__(self, *args, **kwargs):
         self.close()
 
 
-def _process_ctx_request(ctx, args):
+def _process_ctx_request(ctx, args): # pylint: disable=too-many-branches,too-many-statements
     current = ctx
-    num_args = len(args)
     index = 0
+
+    try:
+        # TODO: should there be a way to escape "=" in case it is needed as real argument?
+        equals_index = args.index('=')
```

Comment:
Maybe add a comment here that says that `.index` raises a `ValueError` if the value is not in the list. And that is why you check for `ValueError` in the end. It wasn't clear for @mxmrlv also.
In addition, why does all the other logic under this line is in the `try` if it doesn't expected to raise `ValueError`. I understand that extracting this part will force you to refactor the order of the statements a bit - just a thought =)